Form/Type/ColorSelectorType.php

```diff
@@ -49,6 +49,13 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::YELLOW,
             ),
         ));
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $resolver->setDefaults(array_merge($resolver->getDefinedOptions(), array(
+                'choices' => array_unique(array_flip(Colors::getAll())),
+            )));
+        }
```

Comment:
``choices_as_values``  is [deprecated](http://symfony.com/doc/current/reference/forms/types/choice.html#choices-as-values) in Symfony 3.x. Should I add this option? About the personal code it could be done, just clarify the ``choices_as_values`` option first.